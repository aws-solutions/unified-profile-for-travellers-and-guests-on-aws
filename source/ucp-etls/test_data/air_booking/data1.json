{    "objectVersion": 17,    "modelVersion": "1.0",    "id": "VUP79M",    "externalIds": null,    "lastUpdatedOn": "2021-06-07T07:58:51.88075427Z",    "createdOn": "2021-04-07T07:58:51.88075427Z",    "lastUpdatedBy": "Guiseppe Collier",    "createdBy": "Gavin Bernier",    "creationChannelId": "gds",    "price": {        "status": null,        "grandTotal": 0.0    },    "itinerary": {        "from": "AMS",        "to": "MAD",        "departureDate": "2021-10-27",        "departureTime": "02:18",        "arrivalDate": "2021-10-27",        "arrivalTime": "09:42",        "duration": "7h24m0s",        "segments": [            {                "rank": 0,                "from": "AMS",                "to": "DEN",                "departureDate": "2021-10-27",                "departureTime": "02:18",                "arrivalDate": "2021-10-27",                "arrivalTime": "05:34",                "flightNumber": "DL1317",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            },            {                "rank": 1,                "from": "DEN",                "to": "MCO",                "departureDate": "2021-10-27",                "departureTime": "04:49",                "arrivalDate": "2021-10-27",                "arrivalTime": "07:02",                "flightNumber": "DL2233",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            },            {                "rank": 2,                "from": "MCO",                "to": "MAD",                "departureDate": "2021-10-27",                "departureTime": "08:29",                "arrivalDate": "2021-10-27",                "arrivalTime": "09:42",                "flightNumber": "DL3239",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            }        ],        "inventory": {            "totalSeats": 0,            "inventoryByFareClass": null        },        "status": "tentative"    },    "return": {        "from": "MAD",        "to": "AMS",        "departureDate": "2021-09-08",        "departureTime": "05:49",        "arrivalDate": "2021-09-09",        "arrivalTime": "00:24",        "duration": "18h35m0s",        "segments": [            {                "rank": 0,                "from": "MAD",                "to": "MCO",                "departureDate": "2021-09-08",                "departureTime": "05:49",                "arrivalDate": "2021-09-08",                "arrivalTime": "07:57",                "flightNumber": "DL8568",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            },            {                "rank": 1,                "from": "MCO",                "to": "DEN",                "departureDate": "2021-09-08",                "departureTime": "08:37",                "arrivalDate": "2021-09-08",                "arrivalTime": "09:56",                "flightNumber": "DL7284",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            },            {                "rank": 2,                "from": "DEN",                "to": "AMS",                "departureDate": "2021-09-08",                "departureTime": "12:03",                "arrivalDate": "2021-09-09",                "arrivalTime": "00:24",                "flightNumber": "DL3366",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            }        ],        "inventory": {            "totalSeats": 0,            "inventoryByFareClass": null        },        "status": ""    },    "passengerInfo": {        "passengers": [            {                "modelVersion": "1.0",                "id": "6393037347",                "lastUpdatedOn": "2023-01-29T06:00:11.197550607Z",                "createdOn": "2022-09-29T06:00:11.197550607Z",                "lastUpdatedBy": "Katarina Keebler",                "createdBy": "Anastacio Schaefer",                "emails": [],                "phones": [                    {                        "type": "business",                        "number": "6351736203",                        "primary": false,                        "countryCode": 45                    },                    {                        "type": "personal",                        "number": "532-589-4548",                        "primary": true,                        "countryCode": 37                    },                    {                        "type": "",                        "number": "5231655752",                        "primary": true,                        "countryCode": 99                    }                ],                "addresses": [                    {                        "type": "business",                        "line1": "59890 Lake Pines fort",                        "line2": "more content line 2",                        "line3": "more content line 3",                        "line4": "more content line 4",                        "city": "Durham",                        "stateProvince": {                            "code": "MS",                            "name": ""                        },                        "postalCode": "47882",                        "country": {                            "code": "VI",                            "name": ""                        },                        "primary": false                    }                ],                "honorific": "Mrs",                "firstName": "Stella",                "middleName": "Myrtie",                "lastName": "Dietrich",                "gender": "female",                "pronoun": "",                "dateOfBirth": "1916-07-10",                "language": {                    "code": "he",                    "name": ""                },                "nationality": {                    "code": "DZ",                    "name": ""                },                "jobTitle": "Consultant",                "parentCompany": "GenoSpace",                "loyaltyPrograms": [                    {                        "id": "6060514131",                        "programName": "elite",                        "miles": 170783.0,                        "milesToNextLevel": 3568.0,                        "level": "platinium",                        "joined": "2022-09-29T06:00:11.197550607Z"                    }                ],                "identityProofs": null            },            {                "modelVersion": "1.0",                "id": "7226709910",                "lastUpdatedOn": "2021-08-08T08:03:29.894988695Z",                "createdOn": "2021-04-08T08:03:29.894988695Z",                "lastUpdatedBy": "Patrick Nikolaus",                "createdBy": "Valerie Walter",                "emails": [],                "phones": [                    {                        "type": "business",                        "number": "136.314.7624",                        "primary": true,                        "countryCode": 82                    },                    {                        "type": "personal",                        "number": "878-520-8118",                        "primary": false,                        "countryCode": 64                    }                ],                "addresses": [                    {                        "type": "mailing",                        "line1": "2150 North Causeway mouth",                        "line2": "more content line 2",                        "line3": "more content line 3",                        "line4": "more content line 4",                        "city": "Philadelphia",                        "stateProvince": {                            "code": "AE",                            "name": ""                        },                        "postalCode": "15256",                        "country": {                            "code": "CA",                            "name": ""                        },                        "primary": false                    }                ],                "honorific": "Lady",                "firstName": "Emelie",                "middleName": "Oswaldo",                "lastName": "Barrows",                "gender": "female",                "pronoun": "",                "dateOfBirth": "1999-04-21",                "language": {                    "code": "bn",                    "name": ""                },                "nationality": {                    "code": "SY",                    "name": ""                },                "jobTitle": "Specialist",                "parentCompany": "Porch",                "loyaltyPrograms": [                    {                        "id": "0435270480",                        "programName": "ascend",                        "miles": 26893.0,                        "milesToNextLevel": 6121.0,                        "level": "silver",                        "joined": "2021-04-08T08:03:29.894988695Z"                    },                    {                        "id": "2530789961",                        "programName": "club",                        "miles": 653201.0,                        "milesToNextLevel": 2328.0,                        "level": "bronze",                        "joined": "2021-04-08T08:03:29.894988695Z"                    }                ],                "identityProofs": null            },            {                "modelVersion": "1.0",                "id": "7929069526",                "lastUpdatedOn": "2021-04-20T02:22:24.524202159Z",                "createdOn": "2021-03-20T02:22:24.524202159Z",                "lastUpdatedBy": "Louisa Waters",                "createdBy": "Rosa Paucek",                "emails": [                    {                        "type": "personal",                        "address": "nathanielbrakus@dickens.name",                        "primary": true                    },                    {                        "type": "",                        "address": "matteotorphy@hintz.org",                        "primary": true                    }                ],                "phones": [],                "addresses": [],                "honorific": "Cllr",                "firstName": "Beverly",                "middleName": "Malika",                "lastName": "Stehr",                "gender": "male",                "pronoun": "they",                "dateOfBirth": "1908-02-19",                "language": {                    "code": "wo",                    "name": ""                },                "nationality": {                    "code": "AX",                    "name": ""                },                "jobTitle": "Strategist",                "parentCompany": "Porch",                "loyaltyPrograms": [                    {                        "id": "6583533469",                        "programName": "reward",                        "miles": 551768.0,                        "milesToNextLevel": 9588.0,                        "level": "bronze",                        "joined": "2021-03-20T02:22:24.524202159Z"                    },                    {                        "id": "8388922773",                        "programName": "reward",                        "miles": 728211.0,                        "milesToNextLevel": 9231.0,                        "level": "platinium",                        "joined": "2021-03-20T02:22:24.524202159Z"                    }                ],                "identityProofs": null            },            {                "modelVersion": "1.0",                "id": "3078342134",                "lastUpdatedOn": "2021-05-26T22:44:31.623908935Z",                "createdOn": "2021-03-26T22:44:31.623908935Z",                "lastUpdatedBy": "Abdullah Blick",                "createdBy": "Lamar Walsh",                "emails": [                    {                        "type": "business",                        "address": "zenahirthe@heller.name",                        "primary": true                    }                ],                "phones": [                    {                        "type": "mobile",                        "number": "562-169-5540",                        "primary": true,                        "countryCode": 53                    },                    {                        "type": "mobile",                        "number": "7886690684",                        "primary": true,                        "countryCode": 25                    }                ],                "addresses": [],                "honorific": "",                "firstName": "Hershel",                "middleName": "Ricardo",                "lastName": "Reinger",                "gender": "other",                "pronoun": "",                "dateOfBirth": "1925-11-27",                "language": {                    "code": "an",                    "name": ""                },                "nationality": {                    "code": "ER",                    "name": ""                },                "jobTitle": "Technician",                "parentCompany": "Verdafero",                "loyaltyPrograms": [],                "identityProofs": null            },            {                "modelVersion": "1.0",                "id": "7416069925",                "lastUpdatedOn": "2022-02-06T16:06:28.024163824Z",                "createdOn": "2021-10-06T16:06:28.024163824Z",                "lastUpdatedBy": "Jackson Kertzmann",                "createdBy": "Garnet Schmeler",                "emails": [                    {                        "type": "",                        "address": "abbeycronin@volkman.com",                        "primary": true                    },                    {                        "type": "",                        "address": "justinerussel@emard.name",                        "primary": true                    },                    {                        "type": "",                        "address": "raoulgoyette@fritsch.info",                        "primary": false                    }                ],                "phones": [                    {                        "type": "",                        "number": "4585009949",                        "primary": true,                        "countryCode": 17                    },                    {                        "type": "business",                        "number": "5395171401",                        "primary": false,                        "countryCode": 20                    },                    {                        "type": "home",                        "number": "2133617942",                        "primary": false,                        "countryCode": 22                    }                ],                "addresses": [                    {                        "type": "business",                        "line1": "3866 Hollow shire",                        "line2": "more content line 2",                        "line3": "more content line 3",                        "line4": "more content line 4",                        "city": "San Diego",                        "stateProvince": {                            "code": "NY",                            "name": ""                        },                        "postalCode": "23284",                        "country": {                            "code": "CN",                            "name": ""                        },                        "primary": false                    }                ],                "honorific": "Sir",                "firstName": "Maye",                "middleName": "Amelia",                "lastName": "Gerhold",                "gender": "male",                "pronoun": "he",                "dateOfBirth": "1995-06-18",                "language": {                    "code": "kv",                    "name": ""                },                "nationality": {                    "code": "AG",                    "name": ""                },                "jobTitle": "Architect",                "parentCompany": "FutureAdvisor",                "loyaltyPrograms": [                    {                        "id": "3790802887",                        "programName": "reward",                        "miles": 84377.0,                        "milesToNextLevel": 9472.0,                        "level": "gold",                        "joined": "2021-10-06T16:06:28.024163824Z"                    },                    {                        "id": "8983778121",                        "programName": "club",                        "miles": 39076.0,                        "milesToNextLevel": 4650.0,                        "level": "silver",                        "joined": "2021-10-06T16:06:28.024163824Z"                    }                ],                "identityProofs": null            }        ]    },    "paymentInformation": {        "paymentType": "cash",        "ccInfo": {            "token": "",            "cardType": "",            "cardExp": "",            "cardCvv": "",            "expiration": "",            "name": "",            "address": {                "type": "",                "line1": "",                "line2": "",                "line3": "",                "line4": "",                "city": "",                "stateProvince": {                    "code": "",                    "name": ""                },                "postalCode": "",                "country": {                    "code": "",                    "name": ""                },                "primary": false            }        },        "routingNumber": "",        "accountNumber": "",        "voucherID": "",        "address": {            "type": "shipping",            "line1": "39323 Trace furt",            "line2": "more content line 2",            "line3": "more content line 3",            "line4": "more content line 4",            "city": "Seattle",            "stateProvince": {                "code": "LA",                "name": ""            },            "postalCode": "96299",            "country": {                "code": "LR",                "name": ""            },            "primary": true        }    },    "promocode": "9XDCB12XAZ",    "email": "",    "phone": "",    "status": "tentative"}