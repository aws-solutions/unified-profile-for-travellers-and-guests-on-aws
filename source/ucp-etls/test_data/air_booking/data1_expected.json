{"air_booking_recs": [{"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-JFK-ORD", "from": "JFK", "to": "ORD", "flight_number": "LF3057", "departure_date": "2021-12-30", "departure_time": "15:44", "arrival_date": "2021-12-30", "arrival_time": "20:10", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Eleonore", "middle_name": "Coby", "last_name": "Kling", "gender": "female", "pronoun": "she", "date_of_birth": "2015-08-11", "job_title": "Associate", "company": "Barchart", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "pearliecarter@bailey.info", "email_type": "personal", "phone_business": "8400997949", "phone_type": "mobile", "traveller_id": "0850621616", "address_shipping_line1": "934 Port Ways shire", "address_shipping_line2": "more content line 2", "address_shipping_line3": "more content line 3", "address_shipping_line4": "more content line 4", "address_shipping_postal_codes": "38981", "address_shipping_city": "Dallas", "address_shipping_state_province": "DE", "address_shipping_country": "SG", "address_type": "shipping"}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-ORD-FRA", "from": "ORD", "to": "FRA", "flight_number": "LF3316", "departure_date": "2021-12-30", "departure_time": "18:44", "arrival_date": "2021-12-31", "arrival_time": "01:22", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Eleonore", "middle_name": "Coby", "last_name": "Kling", "gender": "female", "pronoun": "she", "date_of_birth": "2015-08-11", "job_title": "Associate", "company": "Barchart", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "pearliecarter@bailey.info", "email_type": "personal", "phone_business": "8400997949", "phone_type": "mobile", "traveller_id": "0850621616", "address_shipping_line1": "934 Port Ways shire", "address_shipping_line2": "more content line 2", "address_shipping_line3": "more content line 3", "address_shipping_line4": "more content line 4", "address_shipping_postal_codes": "38981", "address_shipping_city": "Dallas", "address_shipping_state_province": "DE", "address_shipping_country": "SG", "address_type": "shipping"}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-FRA-ATL", "from": "FRA", "to": "ATL", "flight_number": "LF8648", "departure_date": "2021-12-30", "departure_time": "21:53", "arrival_date": "2021-12-31", "arrival_time": "04:02", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Eleonore", "middle_name": "Coby", "last_name": "Kling", "gender": "female", "pronoun": "she", "date_of_birth": "2015-08-11", "job_title": "Associate", "company": "Barchart", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "pearliecarter@bailey.info", "email_type": "personal", "phone_business": "8400997949", "phone_type": "mobile", "traveller_id": "0850621616", "address_shipping_line1": "934 Port Ways shire", "address_shipping_line2": "more content line 2", "address_shipping_line3": "more content line 3", "address_shipping_line4": "more content line 4", "address_shipping_postal_codes": "38981", "address_shipping_city": "Dallas", "address_shipping_state_province": "DE", "address_shipping_country": "SG", "address_type": "shipping"}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-ATL-FRA", "from": "ATL", "to": "FRA", "flight_number": "LF0436", "departure_date": "2021-05-01", "departure_time": "17:24", "arrival_date": "2021-05-01", "arrival_time": "21:22", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Eleonore", "middle_name": "Coby", "last_name": "Kling", "gender": "female", "pronoun": "she", "date_of_birth": "2015-08-11", "job_title": "Associate", "company": "Barchart", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "pearliecarter@bailey.info", "email_type": "personal", "phone_business": "8400997949", "phone_type": "mobile", "traveller_id": "0850621616", "address_shipping_line1": "934 Port Ways shire", "address_shipping_line2": "more content line 2", "address_shipping_line3": "more content line 3", "address_shipping_line4": "more content line 4", "address_shipping_postal_codes": "38981", "address_shipping_city": "Dallas", "address_shipping_state_province": "DE", "address_shipping_country": "SG", "address_type": "shipping"}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-FRA-ORD", "from": "FRA", "to": "ORD", "flight_number": "LF6779", "departure_date": "2021-05-01", "departure_time": "18:30", "arrival_date": "2021-05-02", "arrival_time": "00:13", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Eleonore", "middle_name": "Coby", "last_name": "Kling", "gender": "female", "pronoun": "she", "date_of_birth": "2015-08-11", "job_title": "Associate", "company": "Barchart", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "pearliecarter@bailey.info", "email_type": "personal", "phone_business": "8400997949", "phone_type": "mobile", "traveller_id": "0850621616", "address_shipping_line1": "934 Port Ways shire", "address_shipping_line2": "more content line 2", "address_shipping_line3": "more content line 3", "address_shipping_line4": "more content line 4", "address_shipping_postal_codes": "38981", "address_shipping_city": "Dallas", "address_shipping_state_province": "DE", "address_shipping_country": "SG", "address_type": "shipping"}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-ORD-JFK", "from": "ORD", "to": "JFK", "flight_number": "LF2894", "departure_date": "2021-05-01", "departure_time": "19:48", "arrival_date": "2021-05-02", "arrival_time": "06:37", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Eleonore", "middle_name": "Coby", "last_name": "Kling", "gender": "female", "pronoun": "she", "date_of_birth": "2015-08-11", "job_title": "Associate", "company": "Barchart", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "pearliecarter@bailey.info", "email_type": "personal", "phone_business": "8400997949", "phone_type": "mobile", "traveller_id": "0850621616", "address_shipping_line1": "934 Port Ways shire", "address_shipping_line2": "more content line 2", "address_shipping_line3": "more content line 3", "address_shipping_line4": "more content line 4", "address_shipping_postal_codes": "38981", "address_shipping_city": "Dallas", "address_shipping_state_province": "DE", "address_shipping_country": "SG", "address_type": "shipping"}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-JFK-ORD", "from": "JFK", "to": "ORD", "flight_number": "LF3057", "departure_date": "2021-12-30", "departure_time": "15:44", "arrival_date": "2021-12-30", "arrival_time": "20:10", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Claud", "middle_name": "Andreanne", "last_name": "Cummerata", "gender": "other", "pronoun": "he", "date_of_birth": "1909-11-27", "job_title": "Orchestrator", "company": "Social Explorer", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "daisybeer@friesen.com", "email_type": "", "phone": "3284778536", "phone_type": "", "traveller_id": "3047621703", "address_line1": "19446 Port Creek chester", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_postal_codes": "48148", "address_city": "Boise", "address_state_province": "VT", "address_country": "KY", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-ORD-FRA", "from": "ORD", "to": "FRA", "flight_number": "LF3316", "departure_date": "2021-12-30", "departure_time": "18:44", "arrival_date": "2021-12-31", "arrival_time": "01:22", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Claud", "middle_name": "Andreanne", "last_name": "Cummerata", "gender": "other", "pronoun": "he", "date_of_birth": "1909-11-27", "job_title": "Orchestrator", "company": "Social Explorer", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "daisybeer@friesen.com", "email_type": "", "phone": "3284778536", "phone_type": "", "traveller_id": "3047621703", "address_line1": "19446 Port Creek chester", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_postal_codes": "48148", "address_city": "Boise", "address_state_province": "VT", "address_country": "KY", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-FRA-ATL", "from": "FRA", "to": "ATL", "flight_number": "LF8648", "departure_date": "2021-12-30", "departure_time": "21:53", "arrival_date": "2021-12-31", "arrival_time": "04:02", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Claud", "middle_name": "Andreanne", "last_name": "Cummerata", "gender": "other", "pronoun": "he", "date_of_birth": "1909-11-27", "job_title": "Orchestrator", "company": "Social Explorer", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "daisybeer@friesen.com", "email_type": "", "phone": "3284778536", "phone_type": "", "traveller_id": "3047621703", "address_line1": "19446 Port Creek chester", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_postal_codes": "48148", "address_city": "Boise", "address_state_province": "VT", "address_country": "KY", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-ATL-FRA", "from": "ATL", "to": "FRA", "flight_number": "LF0436", "departure_date": "2021-05-01", "departure_time": "17:24", "arrival_date": "2021-05-01", "arrival_time": "21:22", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Claud", "middle_name": "Andreanne", "last_name": "Cummerata", "gender": "other", "pronoun": "he", "date_of_birth": "1909-11-27", "job_title": "Orchestrator", "company": "Social Explorer", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "daisybeer@friesen.com", "email_type": "", "phone": "3284778536", "phone_type": "", "traveller_id": "3047621703", "address_line1": "19446 Port Creek chester", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_postal_codes": "48148", "address_city": "Boise", "address_state_province": "VT", "address_country": "KY", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-FRA-ORD", "from": "FRA", "to": "ORD", "flight_number": "LF6779", "departure_date": "2021-05-01", "departure_time": "18:30", "arrival_date": "2021-05-02", "arrival_time": "00:13", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Claud", "middle_name": "Andreanne", "last_name": "Cummerata", "gender": "other", "pronoun": "he", "date_of_birth": "1909-11-27", "job_title": "Orchestrator", "company": "Social Explorer", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "daisybeer@friesen.com", "email_type": "", "phone": "3284778536", "phone_type": "", "traveller_id": "3047621703", "address_line1": "19446 Port Creek chester", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_postal_codes": "48148", "address_city": "Boise", "address_state_province": "VT", "address_country": "KY", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "booking_id": "E5QEYX", "segment_id": "E5QEYX-ORD-JFK", "from": "ORD", "to": "JFK", "flight_number": "LF2894", "departure_date": "2021-05-01", "departure_time": "19:48", "arrival_date": "2021-05-02", "arrival_time": "06:37", "channel": "gds", "status": "confirmed", "honorific": "Ms", "first_name": "Claud", "middle_name": "Andreanne", "last_name": "Cummerata", "gender": "other", "pronoun": "he", "date_of_birth": "1909-11-27", "job_title": "Orchestrator", "company": "Social Explorer", "price": 0.0, "payment_type": "voucher", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "daisybeer@friesen.com", "email_type": "", "phone": "3284778536", "phone_type": "", "traveller_id": "3047621703", "address_line1": "19446 Port Creek chester", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_postal_codes": "48148", "address_city": "Boise", "address_state_province": "VT", "address_country": "KY", "address_type": ""}], "common_email_recs": [{"model_version": "1.0", "object_type": "email_history", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "address": "pearliecarter@bailey.info", "type": "personal", "traveller_id": "0850621616"}, {"model_version": "1.0", "object_type": "email_history", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "address": "daisybeer@friesen.com", "type": "", "traveller_id": "3047621703"}], "common_phone_recs": [{"model_version": "1.0", "object_type": "phone_history", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "number": "8400997949", "country_code": 52, "type": "mobile", "traveller_id": "0850621616"}, {"model_version": "1.0", "object_type": "phone_history", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "number": "3284778536", "country_code": 5, "type": "", "traveller_id": "3047621703"}, {"model_version": "1.0", "object_type": "phone_history", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "number": "3451974777", "country_code": 43, "type": "business", "traveller_id": "3047621703"}], "air_loyalty_recs": [{"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2021-04-02T14:07:15.895632153Z", "last_updated_by": "Luna Armstrong", "id": "5362558915", "program_name": "elite", "miles": "5362558915", "miles_to_next_level": 8852.0, "level": "platinium", "joined": "2021-12-09T07:59:14.873255297Z", "traveller_id": "3047621703"}]}